src/Campaigns/Migrations/Tables/CreateCampaignsTable.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Give\Campaigns\Migrations\Tables;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ * Creates give_campaigns table
+ */
+class CreateCampaignsTable extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public static function id(): string
+    {
+        return 'give-campaigns-create-give-core-campaigns-table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function title(): string
+    {
+        return 'Create give_campaigns table from core';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp(): string
+    {
+        return strtotime('2024-08-26 00:00:00');
+    }
+
+    /**
+     * @inheritDoc
+     * @throws DatabaseMigrationException
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $table = $wpdb->prefix . 'give_campaigns';
+        $charset = DB::get_charset_collate();
+
+        $sql = "CREATE TABLE $table (
+			id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+			form_id INT NOT NULL,
+			campaign_type VARCHAR(12) NOT NULL DEFAULT '',
```

Comment:
@alaca since this has previously existed (in P2P), could you please document any changes using a `@since/@unreleased` tag in the doc block. I think we need to treat this as a change and not necessarily as something "new".